scripts/classification/classification_utils.py

```diff
@@ -0,0 +1,141 @@
+import gluonnlp
+import numpy as np
+import mxnet as mx
+import pandas as pd
+import matplotlib.pyplot as plt
+from tqdm import tqdm
+import copy
+from mxnet.gluon import nn
+from mxnet.gluon.metric import Accuracy, F1, MCC, PearsonCorrelation, CompositeEvalMetric
+from gluonnlp.models import get_backbone
+from gluonnlp.utils.parameter import clip_grad_global_norm
+from gluonnlp.utils.preprocessing import get_trimmed_lengths
+from gluonnlp.utils.misc import get_mxnet_visible_ctx, grouper, repeat
+from mxnet.gluon.data import batchify as bf
+from mxnet.gluon.data import DataLoader
+from mxnet.lr_scheduler import PolyScheduler
+from gluonnlp.utils import set_seed
+
+__all__ = ['get_task']
+
+
+_FEATURE_COLUMN = {'sst':'sentence',
+                   'sts':['sentence1', 'sentence2'],
+                   'cola':'sentence',
+                   'mrpc':['sentence1', 'sentence2'],
+                   'wnli':['sentence1', 'sentence2'],
+                   'qnli':['sentence1', 'sentence2'],
+                   'qqp':['sentence1', 'sentence2'],
+                   'rte':['sentence1', 'sentence2'],
+                   'mnli':['sentence1', 'sentence2']
+                   }
+_LABEL_COLUMN = { 'sst':'label',
+                  'sts':'score',
+                  'cola':'label',
+                  'mrpc':'label',
+                  'qnli':'label',
+                  'wnli':'label',
+                  'qqp':'label',
+                  'rte':'label',
+                  'mnli':'label',
+                  }
+_METRIC = {'mrpc':['Accuracy', 'f1'],
+           'qqp':['Accuracy', 'f1'],
+           'rte':['Accuracy'],
+           'qnli':['Accuracy'],
+           'sts':['PearsonCorrelation'],
+           'cola':['mcc'],
+           'sst':['Accuracy'],
+           'mnli':['Accuracy'],
+           }
+
+class GlueTask:
+    def __init__(self, task_name, class_num, feature_column, label_column, metric):
+        self.task_name = task_name
+        self.class_num = class_num
+        self.feature_column = feature_column
+        self.label_column = label_column
+        self.metric = metric
+
+
+class SST(GlueTask):

```

Comment:
I would recommend to support loading the input as a pandas dataframe.

Reply:
Ok, I'll change that